packages/frontend/next.config.mjs

```diff
@@ -1,10 +1,12 @@
+
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  reactStrictMode: false,
+  reactStrictMode: true,
   experimental: {
     serverActions: {
       allowedOrigins: ['localhost:3000'],
     },
+    missingSuspenseWithCSRBailout: false,
```

Comment:
apparently we should not use this as it will be removed in future nextjs version

https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout